src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
```

Comment:
The `src` property of `@font-face` should point to a font file (like .woff, .woff2, .ttf, etc.), not a webpage. You need to download the font file from Google Fonts and then use it in your project.